docs/content/manual/_incoming/3045-upgrade-with-bonded-nic.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Upgrade Harvester on node that has bonded NICs for management interface
+---
+
+* Related issues: [#3045](https://github.com/harvester/harvester/issues/3045) [BUG] Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface
+
+## Category: 
+* Upgrade
+
+## Environment Setup
+
+- This is to be done on a Harvester cluster where the NICs were configured to be bonded on install for the management interface
+
+## Verification Steps
+
+1. Install Harvester with at least 2 bonded NICs on install in `balance-tlb`
+
+    ![image](https://user-images.githubusercontent.com/83787952/198138366-472d4432-839a-4d9d-a942-d55fe8f6f6d8.png)
+1. Start an upgrade if the responder has one setup or set up a custom upgrade via the instructions [here](https://docs.harvesterhci.io/v1.1/upgrade/automatic/#prepare-an-air-gapped-upgrade)
+
+## Expected Results
+
+- The upgrade should complete successfully and the Harvester cluster should pass sanity checks
```

Comment:
QU: Do we need to elaborate on what the sanity checks should be?
I know through - https://harvester.github.io/tests/manual/ - we don't really have a "Sanity Checks" section.
I would figure, though, we might want to ensure elements of networking related work - maybe like VLAN 1 check to ensure that's active on the management network too?

Reply:
Hmm. good point. We do have sanity checks, but they are only outlined in the parent page in the manual/vm section. I'll update that.

Reply:
Added link.